File: order-java-micronaut/order-service/src/main/java/order/service/OrderEntity.java
package order.service;

import java.util.UUID;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import javax.persistence.Table;
import javax.validation.constraints.NotNull;

@Entity
@Table(name = "Order")
public class OrderEntity {

    public OrderEntity() {
    }

    public OrderEntity(@NotNull String name) {
        this.name = name;
    }

    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    private Long id;

    @NotNull
    @Column(name = "name", nullable = false, unique = true)
    private String name;

    @NotNull
    @Column(name = "transaction_id", nullable = false, unique = true)
    private UUID transactionId;

    @NotNull
    @Column(name = "transaction_status", nullable = false, unique = true)
    private String transactionStatus;

    public Long getId() {
        return id;
    }

    public void setId(Long id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public UUID getTransactionId() {
        return transactionId;
    }

    public void setTransactionId(UUID transactionId) {
        this.transactionId = transactionId;
    }

    public String getTransactionStatus() {
        return transactionStatus;
    }

    public void setTransactionStatus(String transactionStatus) {
        this.transactionStatus = transactionStatus;
    }

    @Override
    public String toString() {
        return "Order{" + "id=" + id + ", name='" + name + '\'' + '}';
    }
}

File: order-golang/order-service/app/order/OrderEntity.go
package order

type OrderEntity struct {
	ID                int    `bson:"id,omitempty" json:"id,omitempty"`
	Detail            string `bson:"detail,omitempty" json:"detail,omitempty"`
	TransactionStatus string `bson:"transaction_status,omitempty" json:"transaction_status,omitempty"`
	TransactionId     string `bson:"transaction_id,omitempty" json:"transaction_id,omitempty"`
	Created           string `bson:"created,omitempty" json:"created,omitempty"`
	Updated           string `bson:"updated,omitempty" json:"updated,omitempty"`
}
